tcms/kiwi_auth/test_admin.py

```diff
@@ -109,3 +111,47 @@ def test_admin_can_add_new_users(self):
 
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertTrue(get_user_model().objects.filter(username='added-by-admin').exists())
+
+
+class TestGroupAdmin(LoggedInTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.admin = UserFactory()
+        cls.admin.is_superuser = True
+        cls.admin.set_password('admin-password')
+        cls.admin.save()
+
+        cls.group = GroupFactory(name='NewGroupName')
+        cls.group.save()
+
+    def test_should_not_be_allowed_to_change_default_groups_names(self):
+        self.client.login(  # nosec:B106:hardcoded_password_funcarg
+            username=self.admin.username,
+            password='admin-password')
+        groups = Group.objects.filter(name__in=['Administrator', 'Tester'])
+
+        for group in groups:
+            response = self.client.get(reverse('admin:auth_group_change', args=[group.id]))
+            self.assertNotContains(response, 'id="id_name"')
+            self.assertContains(response, 'class="grp-readonly"')
```

Comment:
this is `<div class="grp-readonly>group.name</div>"` which IMO is a better/more explicit match in case we add some more read-only fields in the future.